files/anyjs/update.json

```diff
@@ -0,0 +1,14 @@
+{
+  "packageManager": "github",
+  "name": "anyjs",
+  "repo": "any-js/anyjs",
+  "files": {
+    "basePath": "dist/",
+    "include": [
+      "any.css", "any.min.css",
```

Comment:
Can you just leave the min files?
You can use this rule `*.min.*` to add them all